packages/dumb/src/components/trends/cbg/cbg-date-trace/cbg-date-traces-animation-container.tsx

```diff
@@ -46,7 +46,7 @@ export const CbgDateTracesAnimationContainer: FunctionComponent<CbgDateTracesAni
 
   return (
     <g>
-      {Object.values(data).map((datum: CbgDateTrace[], index: number) => (
+      {[...data.values()].map((datum: CbgDateTrace[], index: number) => (
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏The syntax looks weird here, it should work with `data.map()`